app/scss/indexPage.scss

```diff
@@ -0,0 +1,43 @@
+@import "imports/variables";
+
+* {
+  box-sizing: border-box;
+  body {
+    font-family: 'Asap', sans-serif;
+    .logo {
+      width: 300px;
+      float: right;
+      margin: 2%;
+    }
+    .start-button {
+      width: 110px;
+      background: $primary-button;
+      border: 1px solid #fff;
```

Comment:
We already have color variable for #fff. Please use defined color variable. (Refer to variables.scss)